### PR TITLE
Add cheat sheet entry on `rose date` -> `isodatetime`

### PR DESCRIPTION
**Pull request by MetRonnie**
Paired with https://github.com/cylc/cylc-flow/pull/5838

image

**Requirements check-list**
- [x] I have read `CONTRIBUTING.md` and added my name as a Code Contributor.

### CODE REVIEW

**Comment by MetRonnie** [comment]
(Tests failing on master)

File: src/7-to-8/cheat-sheet.rst [hunk 1]
```diff
@@ -260,6 +260,29 @@ Generate a visualisation for a workflow without running it:
...
+   :class: grid-table
+
+   * - **Cylc 7** & Rose 2019
+     - **Cylc 8** (Rose 2)
+   * - ::
+
+         rose date -c
+     - ::
+
+         isodatetime ref
```
Comment (ukmo-ccbunney) [inline src/7-to-8/cheat-sheet.rst:278]: Just a comment : I like the syntax synopsis as you have defined it in the linter with the square brackets as this makes it clearer that the `ref` is replacing the `-c`, i.e.

`rose date [-c]` => `isodatetime [ref]`

**Comment by MetRonnie** [comment]
Slightly changed for clarity:

image

> @ukmo-ccbunney: Just a comment : I like the syntax synopsis as you have defined it in the linter with the square brackets as this makes it clearer that the `ref` is replacing the `-c`, i.e.
> 
> `rose date [-c]` => `isodatetime [ref]`

This was done for brevity as the `-c`/`ref` is optional. Here, I think it's fairly clear with how the rows line up?

**Comment by ukmo-ccbunney** [comment]
Looks perfect @MetRonnie .

File: src/7-to-8/cheat-sheet.rst [hunk 2]
```diff
@@ -260,6 +260,31 @@ Generate a visualisation for a workflow without running it:
...
+     - **Cylc 8** (Rose 2)
+   * - ::
+
+         rose date <point> --offset <offset>
+     - ::
+
+         isodatetime <point> --offset <offset>
+   * - ::
+
+         rose date -c
```
Comment (oliver-sanders) [inline src/7-to-8/cheat-sheet.rst:281]:
```suggestion
         rose date -c
         # equivalent to:
         rose date "$CYLC_TASK_CYCLE_POINT"
```

**Review by oliver-sanders — APPROVED** [review]

**Pull request merged**